tokens/theme/dark.json

```
@@ -534,7 +534,7 @@
       "$type": "color",
       "$value": "{comete.biscay.500}"
     },
-    "selected-inverted": {
+    "selected inverted": {
```

```suggestion
    "selected-inverted": {
```

La modification casse le build :
```
$ npm run build

> comete-design-tokens@1.0.0 build
> node build.mjs


scss
- dist/base.scss
- dist

scss
✔︎ dist/base.scss
file:///C:/dev/design-tokens/node_modules/style-dictionary/lib/StyleDictionary.js:564
        throw new Error(err);
              ^

Error:
Reference Errors:
Some token references (1) could not be found.

{calendar.cell.color.border.selected-inverted} tries to reference {border.selected inverted}, which is not defined.
```